src/fragments/start/getting-started/vanillajs/setup.mdx

```diff
@@ -159,35 +159,33 @@ amplify init
 When you initialize Amplify you'll be prompted for some information about the app:
 
 ```console
-Enter a name for the project (amplifyjsapp)
-
-# All AWS services you provision for your app are grouped into an "environment"
-# A common naming convention is dev, staging, and production
-Enter a name for the environment (dev)
-
-# Sometimes the CLI will prompt you to edit a file, it will use this editor to open those files.
-Choose your default editor
-
-# Amplify supports JavaScript (Web & React Native), iOS, and Android apps
-Choose the type of app that you're building (javascript)
-
-What JavaScript framework are you using (none)
-
-Source directory path (src)
-
-Distribution directory path (dist)
-
-Build command (npm run-script build)
-
-Start command (npm run-script start)
-
-# This is the profile you created with the `amplify configure` command in the introduction step.
-Do you want to use an AWS profile
+? Enter a name for the project: amplifyjsapp
```

Comment:
What do you think about aligning answers with `#`, like you did with `data-model.mdx`? 

```suggestion
? Enter a name for the project:
# amplifyjsapp
```

Reply:
I don't have a strong preference between the two, but we should align on a common pattern.

Reply:
Not sure on this. I find it hard to scan "did I just get the same SLI output?" with the #'s.